communities/asu-phoenix/community-assessment.md

```diff
@@ -0,0 +1,121 @@
+### Definition
+
+- Students at Arizona State University and the Phoenix Area interested in growing and becoming a part of the tech community. 
+
+### Size
+
+- 20000+ students enrolled at Arizona State University 
+    - 5,650 enrolled in Computing Program
+- 200-300 at large scale hackathons    
```

Comment:
Out of the 200-300 students attending, around 80% of them were ASU students 😄 

And this was the first semester that full blown hackathons were run here! I know at HackAZ, in Tuscon they roughly get 800 students with around 50% or more mainly being AZ Hackers.